LandSandBoat/server
# [Mob] Puk family and mobskill adjustments

**_I affirm:_**

## What does this pull request do?

This makes some adjustments to the Puk family. It primarily does three things:

1. Remove the debunked theory that Puks have regain during wind weather or during windday. I tested this on retail by charming and releasing Puks repeatedly and checking their TP. They never once had their TP increase.
2. Thoroughly tested White Wind. I verified the correct battle message and math which was correct. Additionally, I discovered after an hour of testing that Snaggletooth Peapuks are only able to use White Wind once while the NMs are able to use it multiple times.
3. Lowers TP gain from elemental magic corresponding to day from 3000 to 1000. This was also tested on retail by casting dia repeatedly on lightsday and the mob only using a TP move every third dia.

If retail proof is needed, I have small videos of the Puks regain testing and logs for the White Wind testing.

## Steps to test these changes

To test the White Wind change:

1. `!gotoid 17404239`
2. Use `!tp 3000` repeatedly and see that it only ever will use White Wind once.
